src/runtime_src/core/pcie/linux/system_linux.cpp

```diff
@@ -133,6 +157,10 @@ get_os_info(boost::property_tree::ptree &pt)
       }
       ifs.close();
   }
+
+  pt.put("model", machine_info());
+  pt.put("cores", std::thread::hardware_concurrency());
+  pt.put("memory", sysconf(_SC_PHYS_PAGES) * sysconf(_SC_PAGE_SIZE) / 0x100000);
```

Comment:
Same comment as comment regarding changing the JSON property name and how the value is formatted.